sys/linux/auto-manual.txt.const

```diff
@@ -2,5 +2,11 @@
 arches = 386, amd64, arm, arm64, mips64le, ppc64le, riscv64, s390x
 AF_NETLINK = ???
 NETLINK_GENERIC = ???
+NS_GET_PARENT = ???
+SIOCGIFINDEX = ???
```

Comment:
This doesn't look normal. You're likely missing some includes in the `auto-manual.txt` file.

Reply:
This file is removed now!